backend_python/backend/routes.py

```diff
@@ -84,7 +85,8 @@ def endpoint_start_game(slug):
 @app.route('/api/towns/<slug>/votes', methods = ['POST'])
 def endpoint_vote(slug):
     town = find_town(slug)
```

Comment:
do we operate on `Game` instance in some places and on `Town` instance in others? i believe `find_town` still returns `Town` instances?